tests/__init__.py

```diff
@@ -84,6 +111,81 @@ def setup_package():
     """
     dj.config['safemode'] = False
 
+    # Add old MySQL
+    source = "/src/tests/external-legacy-data"
+    db_name = "djtest_blob_migrate"
+    db_file = "v0_11.sql"
+    conn_root.query("""
+        CREATE DATABASE {};
+        """.format(db_name))
+
+    def parse_sql(filename):
+        data = open(filename, 'r').readlines()
+        stmts = []
+        DELIMITER = ';'
+        stmt = ''
+
+        for lineno, line in enumerate(data):
```

Comment:
`data` is unnecessary. You can loop directly over the text file even more efficiently.